src/Utilities/_DataValidation.php

```diff
@@ -3,21 +3,33 @@
 namespace Momento\Utilities;
 
 use InvalidArgumentException;
+use Momento\Cache\Errors\InvalidArgumentError;
 
 if (!function_exists('validateTtl')) {
-    function validateTtl($ttlSeconds) : void
+    function validateTtl(int $ttlSeconds) : void
     {
         if (!is_int($ttlSeconds) || $ttlSeconds < 0) {
-            throw new InvalidArgumentException("TTL Seconds must be a non-negative integer");
+            throw new InvalidArgumentError("TTL Seconds must be a non-negative integer");
         }
     }
 }
 
 if (!function_exists('validateCacheName')) {
-    function validateCacheName($cacheName) : void
+    function validateCacheName(string $cacheName) : void
     {
         if (!$cacheName || !is_string($cacheName)) {
-            throw new InvalidArgumentException("Cache name must be a non-empty string");
+            throw new InvalidArgumentError("Cache name must be a non-empty string");
+        }
+    }
+}
+
+if (!function_exists('validateOperationTimeout')) {
```

Comment:
why do we need these if wrappers around the functions like this? Sorry I think missed in previous pr. Is this normal php thing?

Reply:
The resource I modeled this after used this pattern. I think it's possible the PHP autoloader may try to process this file multiple times, in which case this would avoid errors about functions already being declared.

Reply:
![image](https://media0.giphy.com/media/jUwpNzg9IcyrK/giphy.gif?cid=ecf05e47hyf0fhkcirpy2pcjxr4oz8la9joikp3x7n66dgk9&rid=giphy.gif&ct=g)